homework/lesson 6/src/ru/otus/java/homework/HomeworkTemplate.java

```diff
@@ -0,0 +1,56 @@
+package ru.otus.java.homework;
+
+import java.util.Scanner;
+
+/**
+ * Заготовка для выполнения ДЗ "Система тестирования".
+ */
+public class HomeworkTemplate {
+    public static void main(String[] args) {
+        // Переменные для хранения количества правильных и неправильных ответов
+        int correctCount = 0, wrongCount = 0;
+        final String correct = "Правильно";
+        final String incorrect = "Неправильно";
+        final String err_answer = "Такого варианта ответа не существует";
```

Comment:
Константы в Java принято писать большими буквами:
```suggestion
        final String CORRECT = "Правильно";
        final String INCORRECT = "Неправильно";
        final String ERR_ANSWER = "Такого варианта ответа не существует";
```